packages/web-twig/src/Resources/components/TextArea/stories/TextAreaV1DisabledFeature.twig

```diff
@@ -1,22 +1,15 @@
-<div class="spirit-v1-box-field-disabled docs-FormFieldGrid">
-    <h2 class="docs-Heading">V1 Feature &mdash; new disabled state</h2>
-
-    <TextArea
-        id="textareaDisabledFeature"
-        isDisabled
-        isRequired
-        label="Label"
-        name="textareaDisabledFeature"
-        placeholder="Placeholder"
-    />
-    <TextArea
-        helperText="Message"
-        id="textareaDisabledFilledFeature"
-        isDisabled
-        isRequired
-        label="Label"
-        name="textareaDisabledFilledFeature"
-        placeholder="Placeholder"
-        value="Filled"
-    />
-</div>
+<TextArea
+    id="textareaDisabledV1"
+    isDisabled
+    label="Label"
+    name="textareaDisabledV1"
+    placeholder="Placeholder"
+/>
+<TextArea
+    id="textareaDisabledFilledV1"
+    isDisabled
+    label="Label"
+    name="textareaDisabledFilledV1"
+    placeholder="Placeholder"
+    value="Filled"
+/>
```

Comment:
In `web`, this one is `required`:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/33101ec9-d7a9-4900-9ac0-bd99bc615cb2)